src/vivarium_cluster_tools/vipin/log_parser.py

```diff
@@ -10,82 +9,68 @@
 import requests
 
 
-class WorkerLog:
-
+class PerformanceLog:
     def __init__(self, file: Path):
         self.file = file
-        name = file.stem.split('.')  # TODO: validate name is <sge job id>.<task id>.log
-        self.sge_job_id = int(name[0])
-        self.task_id = int(name[1])
+        self.count = 0
 
     def get_summaries(self) -> dict:
-        """Get all performance summary log messages in WorkerLog"""
-        telemetry_pattern = re.compile(r'^\{\"host\".+\"job_number\".+\}$')
+        """Get all performance summary log messages in PerformanceLog"""
+        telemetry_pattern = re.compile(r'^{\"host\".+\"job_number\".+}$')
 
-        # Ideally we'd only need to look at the tail of the file, but because a worker can do multiple draws, we need
-        # to iterate over all the lines...
-        # TODO: either change worker to cut a new log for each draw and check tail reversed(list(log))
-        #  *OR* parallelize this
+        self.count = 0
         f = self.file.open('r')
```

Comment:
You always want to use a context manager pattern for file opening (and pretty much every other situation that looks like 1. acquire resource, 2. use resource, 3. free resource).  

```
with self.file.open() as f:  # Will default open for read
   ... do stuff with f ...
# Exiting the block guarantees things are cleaned up
```